nodeinfo/manager.go

```diff
@@ -71,43 +82,55 @@ func init() {
 }
 
 // NewInfoManager new info manager
-func NewInfoManager(cfg *Config, t transmitter, h chain, privKey crypto.PrivateKey) *InfoManager {
+func NewInfoManager(cfg *Config, t transmitter, h chain, privKey crypto.PrivateKey, fun delegatesGetFunc) *InfoManager {
 	dm := &InfoManager{
-		nodeMap:     lru.New(cfg.NodeMapSize),
-		transmitter: t,
-		chain:       h,
-		privKey:     privKey,
-		version:     version.PackageVersion,
-		address:     privKey.PublicKey().Address().String(),
-	}
-
-	dm.broadcastTask = routine.NewRecurringTask(func() {
+		nodeMap:      lru.New(cfg.NodeMapSize),
+		transmitter:  t,
+		chain:        h,
```

Comment:
what does `h` mean

Reply:
I forget to change name of this param after rename its type name